### DISCUSSION TITLE
Issue with rendering AnimatedLayout

### DISCUSSION THREAD

**Discussion by JonathanHoWeiXin** [software-mansion/react-native-reanimated#2174] [Q&A]
Having issues with AnimatedLayout, seems to be an issue with rendering, has anyone encountered a similar problem? 

![image](https://user-images.githubusercontent.com/20395269/124394002-ea5b8580-dd2f-11eb-8c8f-49b4dde90930.png)

![image](https://user-images.githubusercontent.com/20395269/124394012-f5aeb100-dd2f-11eb-8cc2-2732700aa15c.png)

**Comment by jgonet** [comment]
Not sure if it helps, but you have a typo in "style" inside CustomFunctionComponent.
